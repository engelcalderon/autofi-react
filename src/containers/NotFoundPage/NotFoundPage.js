import React from 'react';

import css from './NotFoundPage.module.css';

const NotFoundPage = () => {
    return (
        <div className={css.root}>
            <div className={css.content}>
                <div className={css.number}>404</div>
                <h1 className={css.heading}>
                    Sorry, we couldn't find that page.
                </h1>
                <p className={css.description}>
                    We can't find the page you're looking for. Make sure you've typed in the URL correctly.
                </p>
            </div>
        </div>
    );
}

export default NotFoundPage;