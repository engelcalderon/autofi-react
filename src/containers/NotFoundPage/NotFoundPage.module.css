.root {
    display: flex;
    flex-direction: column;
    align-items: center;
}
  
  .content {
    max-width: 600px;
    width: 80%;
    margin-top: 60px;
  }
  
  .number {
    text-align: center;
  }
  
  .heading {
    text-align: center;
    margin-top: 22px;
  }
  
  .description {
    text-align: center;
    margin-top: 24px;
  }