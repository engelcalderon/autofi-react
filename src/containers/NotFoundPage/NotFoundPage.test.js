import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import NotFoundPage from './NotFoundPage';

describe('NotFoundPage', () => {
    it('matches snapshot', () => {
        const props = {
        }
        const tree = toJson(mount(<NotFoundPage {...props} />), { mode: 'deep' });
        expect(tree).toMatchSnapshot();
    })
});