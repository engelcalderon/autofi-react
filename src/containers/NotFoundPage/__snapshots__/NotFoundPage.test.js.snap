// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage matches snapshot 1`] = `
<div
  className="root"
>
  <div
    className="content"
  >
    <div
      className="number"
    >
      404
    </div>
    <h1
      className="heading"
    >
      Sorry, we couldn't find that page.
    </h1>
    <p
      className="description"
    >
      We can't find the page you're looking for. Make sure you've typed in the URL correctly.
    </p>
  </div>
</div>
`;
